backend/src/index.ts

```diff
@@ -271,7 +278,22 @@ const main = async () => {
   app.use(requestErrorHandler);
 
   const server = app.listen(port, async () => {
-    logger.info(`Server started listening at port ${port}`);
+    if (!serverCfg.initialized) {
+      logger.info(`Welcome to Infisical
+
+Create your Infisical administrator account at:
+http://localhost:${port}/admin/signup
```

Comment:
These ports won't be useful because the port that the server is listening on will likley be different than the one that is sending traffic to it